pkg/testcase/node.go

```diff
@@ -16,10 +16,17 @@ func TestNodeStatus(
 	nodeAssertReadyStatus assert.NodeAssertFunc,
 	nodeAssertVersion assert.NodeAssertFunc,
 ) {
+	timeout := "300s"
+	interval := "30s"
 	cluster := factory.GetCluster(GinkgoT())
-	fmt.Println("\nFetching node status")
+	expectedNodeCount := cluster.NumServers + cluster.NumAgents 
+
+	if cluster.ProductType == "rke2" {
```

Comment:
isnt better get in a between range ? i really think all those extra code only for this , kind of unnecessary

Reply:
I need to understand the request here bit better, we can discuss offline

Reply:
updated